install/playbooks/main.yml

```diff
@@ -8,7 +8,7 @@
 # Prepare the system, get its external IP address,
 # and set up remote access
 - import_playbook: system-prepare.yml
-- import_playbook: external-ip.yml
+- import_playbook: external-ip-type.yml
```

Comment:
If the playbook is removed, this is not needed anymore. Each playbook will have a dependency on the common `external-ip-type` role.

Reply:
Yes, we can use the meta on this.